stack/tests/test_bootstrap.py

```diff
@@ -0,0 +1,70 @@
+"""Test bootstrap handler."""
```

Comment:
This could definitely be improved but that's a good start (better than nothing 😄 )